ci-operator/config/openshift/cluster-api-provider-baremetal/openshift-cluster-api-provider-baremetal-master.yaml

```diff
@@ -41,6 +41,16 @@ tests:
   commands: make vet
   container:
     from: src
+- as: manifests
+  commands: |
+    curl -s https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh | sh
+    PATH=.:$PATH make manifests
+  container:
+    from: src
+- as: code-gen
+  commands: make generate
```

Comment:
In other places we have had to introduce separate steps to verify that a command like `make generate` doesn't change anything, to ensure that all changes are checked into git. Does the Makefile in the capbm repo do that test for the `generate` target or do we need a different target?

Reply:
+1. Will work on it as part of a separate overhaul of Makefile targets PR.